### PR TITLE
chore: adapter porting nits

### PR DESCRIPTION
**Pull request by millsp**
* Fixes integration tests
* Removes `tsup` in favor of consistency (we can have a discussion about `tsup`)
* Removes unnecessary `.gitignore`
* Removes extra test in `adapter-libsql`
* ~~Changes `debug` for `@prisma/debug` in adapters~~

/integration

### CODE REVIEW

**Review by Jolg42 — APPROVED** [review]

File: packages/adapter-planetscale/src/planetscale.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/require-await */
 import type planetScale from '@planetscale/database'
+import { Debug } from '@prisma/debug'
```
Comment (janpio) [inline packages/adapter-planetscale/src/planetscale.ts:3]: I see you decided to import `Debug` directly from `@prisma/debug`.
A benefit of the previous solution via `@prisma/driver-adapter-utils` was that `Debug` would also be available for any third party that develops a driver adapter and only has the utils as a dependency.

Worth importing via utils?

**Comment by janpio** [comment]
I see you decided to import Debug directly from @prisma/debug.
A benefit of the previous solution via @prisma/driver-adapter-utils was that Debug would also be available for any third party that develops a driver adapter and only has the utils as a dependency.

Worth importing via utils?

**Comment by millsp** [comment]
When I did this, I was thinking that it would be nice to use our own `debug` package for the consolidated crash reports with the last logs. Now thinking in retrospective, I am not even sure these would be triggered via a wasm engine. I'll restore it as it was.

**Review by jkomyno — APPROVED** [review]
Nice! Please make sure this PR doesn't possibly interfere with the fixes in https://github.com/prisma/prisma-engines/pull/4398 before merging. I can also do that if needed.

**Comment by jkomyno** [comment]
In order to fix releases as well, we probably need to add `repository.url` fields to the newly added `*adapter*` packages (see [CI/CD failure](https://github.com/prisma/prisma/actions/runs/6691046387/job/18177517840#step:5:472)).

**Comment by janpio** [comment]
@Jolg42 might know more about the provenance stuff.

**Comment by Jolg42** [comment]
Right, that should be easy to fix.

**Comment by Jolg42** [comment]
Here it is: https://github.com/prisma/prisma/pull/21683

**Review by janpio — CHANGES_REQUESTED** [review]
Decision and potential change on this left:
> When I did this, I was thinking that it would be nice to use our own debug package for the consolidated crash reports with the last logs. Now thinking in retrospective, I am not even sure these would be triggered via a wasm engine. I'll restore it as it was.

**Pull request merged**